### PR TITLE
Modify how program writes to storage

### PR DESCRIPTION
**Pull request by Kangxun**
Some person attributes are not relevant to CallMeMaybe. "Done" field needs to be added to person attributes.

Correcting fields that are written to storage help to modify the program to suit CallMeMaybe.

Let's,

    introduce a minimal Done command
    create a new field called IsDone
    refactor code to make use of the new field IsDone

Adding a new IsDone field is aligned with the current implementation of having a class for each Person attribute. This helps with reusability of code and readability.

### CODE REVIEW

**Comment by Kangxun** [comment]
Now it writes the 'name', 'phone', 'email' and 'done' fields.

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public PersonCard(Person person, int displayedIndex) {
         phone.setText(person.getPhone().value);
         email.setText(person.getEmail().value);
 
-        if (person.getIsDone()) {
+        if (person.getIsDone().toString() == "True") {
```
Comment (houtenteo) [inline src/main/java/seedu/address/ui/PersonCard.java:55]: Strings are not primitive so they should be compared using the .equals method instead. 
[can see this link if u feel like it](https://stackoverflow.com/questions/7375427/comparing-two-identical-strings-with-returns-false)

**Review by houtenteo — DISMISSED** [review]
Ok can, looks great overall. Just one comment above.

**Review by houtenteo — APPROVED** [review]
Ok very nice.

**Pull request merged**